Watch Extension/AddFeedingView.swift

```diff
@@ -75,21 +76,45 @@ struct AddFeedingView: View {
                         // TODO: These numbers are not exact, figure this out better
                         .offset(x: isAdding(feeding: self.feedingIntent) ? -45 : -metrics.size.width * 0.5 - 50)
                         .gesture(TapGesture().onEnded {
-                            self.store.send(.feeding(.start(type: self.feedingIntent, side: .left)))
-                            self.isShowingSheet = false
+                            
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jasonzurita/BabyPatterns/blob/c7106bd91c271d949fbb3c9cc499512408cb0c64/Watch Extension/AddFeedingView.swift#L79"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>AddFeedingView.swift:79</code></td>
    </tr>
  </tbody>
</table>